foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -49,7 +49,7 @@ Back in the `index.html` lets add the `<html>` element by typing out its opening
 
 ### Head Element
 
-The `<head>` element is where we put important meta-information **about** our webpages, and stuff required for our webpages to render correctly in the browser. Anything included within the head element will **not** be displayed to the user.
+The `<head>` element is where we put important meta-information **about** our webpages, and stuff required for our webpages to render correctly in the browser.
```

Comment:
I like the idea behind this change, but I don't like simply removing this sentence. Do you feel like you could reword it to be more accurate, while also emphasizing that elements within the `head` element *should not* be displayed to the user?

Reply:
When I raised the PR I was not able to think proper replacement for the removed statement but now after thinking some more time I can reword the correct statement.